bindings/python/pymongocrypt/explicit_encrypter.py

```diff
@@ -49,6 +51,26 @@ def __init__(self, master_key=None, key_alt_names=None):
                 requests to. May include port number, e.g.
                 "kms.us-east-1.amazonaws.com:443".
 
+            If the `kms_provider` is "azure" it is required and has the
+            following fields::
+
+              - `keyVaultEndpoint` (string): Host with optional port, e.g.
+                 "example.vault.azure.net".
```

Comment:
Nit: suggest moving keyVaultEndpoint to the end to be consistent with aws "endpoint" and gcp "endpoint". Also it's a bit odd that this field is named keyVaultEndpoint instead of just "endpoint". Is there a reason for that?

Reply:
I have not placed it as you have suggested because for azure this is a required field, whereas for other KMS providers it is optional (generally we list required options before listing the optional ones). 
As for why the field is named `keyVaultEndpoint` and not `endpoint`, I just used the same name that we intend to pass to `libmongocrypt` - this is also the name for this option in the spec. @kevinAlbs should be able to shed some light on why it was so named.

Reply:
Okay, can you add "Required." for this option?

Reply:
Done.